content/docs/command-reference/get.md

```diff
@@ -40,7 +40,7 @@ The `path` argument is used to specify the location of the target to be
 downloaded within the source repository at `url`. `path` can specify any file or
 directory in the source repo, including those tracked by DVC, or by Git. Note
 that DVC-tracked targets should be found in a
-[DVC-file](/doc/user-guide/dvc-file-format) of the project.
+[`.dvc` file](/doc/user-guide/dvc-file-format) of the project.
```

Comment:
```suggestion
[`.dvc` file or `dvc.yaml`](/doc/user-guide/dvc-file-format) of the project.
```

Reply:
Actually no, this should read "... found in `dvc.yaml` or in a `.dvc` file ..." please.

Reply:
Please link both even if it's to the same page for now, I'll fix it later.